### PR TITLE
Enable dotnet-ci build

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
@dotnet-bot test ci please

**Comment by pranavkm** [comment]
@dotnet-bot test ci please

**Comment by pranavkm** [comment]
Merging this in - the build bot requires the groovy files to be present in the dev branch to build

File: build/buildpipeline/linux.groovy [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@Library('dotnet-ci') _
+
+simpleNode('Ubuntu16.04', 'latest-or-auto-docker') {
+    stage ('Checking out source') {
+        checkout scm
+    }
+    stage ('Build') {
+        sh './build.sh --ci'
+    }
+}
```
Comment (bricelam) [inline build/buildpipeline/linux.groovy]: 🤢 Where do I file an issue to switch to YAML? :trollface:

Comment (Eilon) [reply build/buildpipeline/linux.groovy]: Beggars can't be choosers 😉

File: build/buildpipeline/windows.groovy [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@Library('dotnet-ci') _
+
+// 'node' indicates to Jenkins that the enclosed block runs on a node that matches
+// the label 'windows-with-vs'
+simpleNode('Windows_NT','latest') {
+    stage ('Checking out source') {
+        checkout scm
+    }
+    stage ('Build') {
+        bat '.\\run.cmd -CI default-build'
```
Comment (pranavkm) [inline build/buildpipeline/windows.groovy]: @bricelam should I specify these two envs that are used in your appveyor.yml:

```
    Test__SqlServer__DefaultConnection: Server=(local)\SQL2016;Database=master;User ID=sa;Password=Password12!
    Test__SqlServer__SupportsMemoryOptimized: true
```

Comment (smitpatel) [reply build/buildpipeline/windows.groovy]: AppVeyor use SqlServer instance over tcp/ip therefore we defined connection string like this.
Does the machine running this have localdb installed?
If yes, then no need to set env var.

**Pull request merged**